### PR TITLE
New NSF NCAR Branding

### LINKED ISSUE #540: Update assets to comply with the New NSF Branding Theme
Update documentation site, pages, etc. to reflect the NSF branding policy changes (e.g. NSF NCAR) whereever applicable, e.g. logos,  text, etc.

### PR DESCRIPTION
**Pull request by cyschneck**
## PR Summary

Closes #540

## PR Checklist

**General**
- [x] Make an issue if one doesn't already exist
- [x] Link the issue this PR resolves by adding `closes #XXX` to the PR description where XXX is the number of the issue.
- [x] Add a brief summary of changes to `docs/release-notes.rst` in a relevant section for the next unreleased release. Possible sections include: Documentation, New Features, Bug Fixes, Internal Changes, Breaking Changes/Deprecated
- [x] Add appropriate labels to this PR
- [x] Make your changes in a forked repository rather than directly in this repo
- [x] Open this PR as a draft if it is not ready for review
- [x] Convert this PR from a draft to a full PR before requesting reviewers
- [x] Passes `precommit`. To set up on your local, run `pre-commit install` from the top level of the repository. To manually run pre-commits, use `pre-commit run --all-files` and re-add any changed files before committing again and pushing.
- [x] If needed, squash and merge PR commits into a single commit to clean up commit history

### CODE REVIEW

**Comment by cyschneck** [comment]
readthedocs has already been updated to include the new NSF logo, these changes add NSF the few remaining instances in the text

I was fixing some Linux issues on my end so when I committed the changes to release-notes and it appears to have duplicated the original commit for some reason, either way, the changes don't conflict

**Review by anissa111 — CHANGES_REQUESTED** [review]
We should also update [docs/index.rst](https://github.com/NCAR/geocat-comp/blob/1f14139bd63e746aa1cb229d8a6ee8d071cfbaa6/docs/index.rst#L15) (line 15) and [docs/conf.py](https://github.com/NCAR/geocat-comp/blob/1f14139bd63e746aa1cb229d8a6ee8d071cfbaa6/docs/conf.py#L258)

Also, don't forget to include this in `docs/release-notes.rst`!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -43,12 +43,12 @@ Project administrators are responsible for clarifying the standards of acceptabl
 
 ## Reporting
 
-Instances of unacceptable behavior can be brought to the attention of the project administrator(s) who may take any action as outlined in the Consequences section below. However, making a report to a project administrator is not considered an ‘official report’ to UCAR.
+Instances of unacceptable behavior can be brought to the attention of the project administrator(s) who may take any action as outlined in the Consequences section below. However, making a report to a project administrator is not considered an ‘official report’ to NSF UCAR.
```
[the remark this answers is not in the document]
Comment (cyschneck) [reply CODE_OF_CONDUCT.md:46]: No changes needed on the text side of things?

File: CODE_OF_CONDUCT.md
[the remark this answers is not in the document]
Comment (kafitzgerald) [reply CODE_OF_CONDUCT.md:1]: The UCAR Codes of Conduct where this came from live here: https://www.ucar.edu/who-we-are/ethics-integrity/codes-conduct

File: docs/citation.rst [hunk 2]
```diff
@@ -23,20 +23,20 @@ the following text:
...
-    Boulder, CO, USA: UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
+    Boulder, CO, USA: NSF UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
 
 In the above citation text, update the year, GeoCAT-comp version, and DOI as appropriate. For
 example:
 
     **Visualization & Analysis Systems Technologies. (2022).
     Geoscience Community Analysis Toolkit: GeoCAT-comp (v2022.04.0) [Software].
-    Boulder, CO, USA: UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607349.**
+    Boulder, CO, USA: NSF UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607349.**
```
Comment (kafitzgerald) [inline docs/citation.rst:39]:
```suggestion
    Boulder, CO, USA: UCAR/NSF NCAR - Computational and Information Systems Lab. doi:10.5281/zenodo.6607349.**
```

File: docs/citation.rst [hunk 3]
```diff
@@ -23,20 +23,20 @@ the following text:
...
     Geoscience Community Analysis Toolkit: GeoCAT-comp [Software].
-    Boulder, CO: UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607205.**
+    Boulder, CO: NSF UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607205.**
 
 Instead, if you would like to cite a specific version of GeoCAT-comp, use the following text:
 
     **Visualization & Analysis Systems Technologies. (\<Year\>).
     Geoscience Community Analysis Toolkit: GeoCAT-comp (v\<version\>) [Software].
-    Boulder, CO, USA: UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
+    Boulder, CO, USA: NSF UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
```
Comment (kafitzgerald) [inline docs/citation.rst:32]:
```suggestion
    Boulder, CO, USA: UCAR/NSF NCAR - Computational and Information Systems Lab. doi:\<DOI\>.**
```

File: docs/citation.rst [hunk 4]
```diff
@@ -23,20 +23,20 @@ the following text:
 
     **Visualization & Analysis Systems Technologies. (2019).
     Geoscience Community Analysis Toolkit: GeoCAT-comp [Software].
-    Boulder, CO: UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607205.**
+    Boulder, CO: NSF UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607205.**
```
Comment (kafitzgerald) [inline docs/citation.rst:26]:
```suggestion
    Boulder, CO: UCAR/NSF NCAR - Computational and Information Systems Lab. doi:10.5281/zenodo.6607205.**
```

File: CODE_OF_CONDUCT.md
[the remark this answers is not in the document]
Comment (kafitzgerald) [reply CODE_OF_CONDUCT.md:1]: It looks like it's all UCAR so no change needed.

**Review by kafitzgerald — CHANGES_REQUESTED** [review]
Thanks for doing this!

File: docs/citation.rst (hunk 2 above)
Comment (kafitzgerald) [reply docs/citation.rst:39]: I realize the lab name typo is from before, but we might as well fix it.

File: CODE_OF_CONDUCT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anissa111) [reply CODE_OF_CONDUCT.md:46]: I don't think so, but I'd check the source we pulled this from to see what they've done, I think @kafitzgerald linked it below somewhere

File: docs/citation.rst (hunk 2 above)
Comment (anissa111) [reply docs/citation.rst:39]: Actually, might this actually supposed to be UCAR/NSF NCAR?

Comment (kafitzgerald) [reply docs/citation.rst:39]: Is that not in the suggestion as well?  

It does look like they don't love slashes in the new guidelines either (just looked again).  Given that CISL is part of NSF NCAR, I think `NSF NCAR Computational and Information Systems Lab` is probably what this should be.

Last I looked, VAST is Visualization and Analysis Software and Technology as well and not what we have here.  I think that part is in the DOI metadata though as well and should probably be addressed separately.

Sorry for the sidebar here.

Comment (anissa111) [reply docs/citation.rst:39]: Oh gosh, yeah, you're right sorry I was reading the suggested change incorrectly 👍

File: CODE_OF_CONDUCT.md
[the remark this answers is not in the document]
Comment (cyschneck) [reply CODE_OF_CONDUCT.md:1]: Updated to remove the UCAR parts

File: docs/citation.rst (hunk 2 above)
Comment (cyschneck) [reply docs/citation.rst]: `NSF NCAR Computational and Information Systems Lab` seems right based on the branding details

File: docs/index.rst [hunk 5]
```diff
@@ -11,8 +11,8 @@
               wrf, python, wrf-python, weather research and forecasting,
               weather research and forecasting model, model, weather,
               numerical weather prediction, model, matplotlib, cartopy,
-              wrf-arw, arw, ncar, ucar, cisl, ncl, ncar command language,
-              national center for atmospheric research,
+              wrf-arw, arw, ncar, nsf, nsf ncar, ucar, cisl, ncl,
+              ncar command language, national center for atmospheric research,
```
Comment (anissa111) [inline docs/index.rst:15]:
```suggestion
              ncar command language, nsf national center for atmospheric research,
```

I think we need to change this, as well [according to the branding guidance](https://sundog.ucar.edu/page/7480)

**Review by anissa111 — CHANGES_REQUESTED** [review]
We still need to make a change here: https://github.com/NCAR/geocat-comp/blob/1f14139bd63e746aa1cb229d8a6ee8d071cfbaa6/docs/conf.py#L258

File: docs/index.rst (hunk 5 above)
Comment (cyschneck) [reply docs/index.rst:15]: Yep, good catch

**Review by kafitzgerald — COMMENTED** [review]
Looks good to me!

I did reach out to design yesterday to double check on the logos after a quick conversation w/ Orhan about the same thing on a different repo.  Apparently the NSF/GeoCAT logo (https://github.com/NCAR/geocat-comp/blob/main/docs/_static/images/logos/GeoCAT_nsf.svg) we have in the upper left hand corner of the docs (and maybe elsewhere) does need to get replaced.  Sorry if I suggested otherwise.  

The recommendation from design was to use the [new NSF NCAR logo displayed on the Sundog page](https://sundog.ucar.edu/page/7480#Interim-Guidance-on--11) instead.

Let me know if you'd like me to make that change.

**Comment by cyschneck** [comment]
I don't see anything in the branding information about crediting NSF as the docs do in via the [nsf.png](https://github.com/NCAR/geocat-comp/blob/main/docs/_static/images/logos/nsf.png), so can this one remain the same?

**Review by kafitzgerald — APPROVED** [review]

**Review by anissa111 — APPROVED** [review]

**Pull request merged**